lib/game/configureGamePass.js

```diff
@@ -0,0 +1,100 @@
+// Includes
+const http = require('noblox.js/lib/util/http').func
+const getGeneralToken = require('noblox.js/lib/util/getGeneralToken').func
+
+// Args
```

Comment:
Document arguments and return type, see other files that have jsdoc documentation.